src/constants.h

```diff
@@ -62,6 +62,10 @@
 #define INITIAL_VIRTUAL_PID  40000
 #define MAX_VIRTUAL_PID  400000000
 
+// NEEDED FOR STRINGIFY(DEFAULT_PORT)
+#define QUOTE(arg) #arg
+#define STRINGIFY(arg) QUOTE(arg)
```

Comment:
Why not just delete `QUOTE` since we don't seem to be using it elsewhere in the code?